g11n-ws/vip-manager-lite-i18n/src/main/java/com/vmware/vip/core/Interceptor/LiteAPIValidationInterceptor.java

```diff
@@ -113,6 +109,7 @@ public void validate(HttpServletRequest request) throws ValidationException {
 		validateComponent(request);
 		validateComponents(request);
 		validateKey(request);
+		validateKeys(request);
```

Comment:
Can here call the 'validateKeys(HttpServletRequest request)' method in 'g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/validation/ParameterValidation.java'?

In that case you needn't write the same method in two places.

Reply:
yes, the lite can't include the md-service-authen module